pageserver/src/http/routes.rs

```diff
@@ -687,7 +687,14 @@ async fn get_lsn_by_timestamp_handler(
         LsnForTimestamp::Past(lsn) => (lsn, "past"),
         LsnForTimestamp::NoData(lsn) => (lsn, "nodata"),
     };
-    json_response(StatusCode::OK, Result { lsn, kind })
+    let result = Result { lsn, kind };
+    tracing::info!(
+        lsn=?result.lsn,
+        kind=result.kind,
+        timestamp = timestamp_raw.into_owned(),
```

Comment:
```suggestion
    tracing::info!(
        lsn=?result.lsn,
        kind=%result.kind,
        timestamp=%timestamp_raw,
```

- unified style
- I don't see why it would need to be owned?